core/src/SYCL/Kokkos_SYCL_UniqueToken.hpp

```diff
@@ -89,8 +89,10 @@ class UniqueToken<SYCL, UniqueTokenScope::Global> {
     const Kokkos::pair<int, int> result =
         Kokkos::Impl::concurrent_bitset::acquire_bounded(
             m_buffer, m_count
-            // FIXME_SYCL clock_tic doesn't work yet
-            //, Kokkos::Impl::clock_tic() % m_count
+#if !defined(KOKKOS_ENABLE_SYCL) || defined(KOKKOS_ARCH_INTEL_GEN)
```

Comment:
```suggestion
#if defined(KOKKOS_ARCH_INTEL_GEN)
```

Reply:
Also do we actually still need to guard the trailing argument?

Reply:
Yes, `clock_tick` doesn't compile otherwise and I don't see any fallback to use.